src/client/table_client.rs

```diff
@@ -2374,3 +2414,181 @@ impl Default for Builder {
         Self::new()
     }
 }
+
+pub struct ObTableAggregation {
+    table_query: ObTableClientQueryImpl,
+    /// this message is used to record the aggregation order and the corresponding aggregation name
+    message: Vec<String>,
+}
+
+impl ObTableAggregation {
+    fn new(table_name: &str, client: Arc<ObTableClientInner>) -> Self {
+        Self {
+            table_query: ObTableClientQueryImpl::new(table_name, client),
+            message: Vec::new(),
+        }
+    }
+
+    pub fn max(mut self, column_name: String) -> Self {
+        let column_bak= column_name.clone();
+        self.table_query = self.table_query.add_aggregation(ObTableAggregationType::MAX, column_name);
+        self.message.push("max(".to_owned() + &*column_bak + &*")".to_owned());
+        self
+    }
+
+    pub fn min(mut self, column_name: String) -> Self {
+        let column_bak= column_name.clone();
+        self.table_query = self.table_query.add_aggregation(ObTableAggregationType::MIN, column_name);
+        self.message.push("min(".to_owned() + &*column_bak + &*")".to_owned());
```

Comment:
Check available and turn it into:
```suggestion
        self.message.push(format!("min({column_bak:?})"));
```

Reply:
fixed
use format!("max({})", column_bak);

Reply:
I am not sure ``format!("max({})", column_bak);`` could pass the test or not.
Pls try run
```
Cargo fmt
Make Clippy
``` after you finish your code.

Reply:
done.
use format!("min({column_bak})"))